From feecc6478855edbdd34506b283de535bab90e9b6 Mon Sep 17 00:00:00 2001
From: Sven Nierlein <sven@nierlein.de>
Date: Tue, 6 Feb 2018 19:49:33 +0100
Subject: [PATCH] fix typo in disablePanelTitle parameter

Right now, it was not possible to use this parameter  because of a typo in disablePanelTitel.
This patches changes all disablePanelTitel into disablePanelTitle
---
 CHANGELOG.md     | 4 ++--
 histou/basic.php | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index d654de1..84c2314 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -12,7 +12,7 @@
 
 ## v0.4.0 - 30.11.2016
 ### Features
-- URL/Config param to disable paneltitles for bigger graphs. Url flag: disablePanelTitel - works now also on simple templates.
+- URL/Config param to disable paneltitles for bigger graphs. Url flag: disablePanelTitle - works now also on simple templates.
 - URL param to specify a certain template. Take a look at the [README](https://github.com/Griesbacher/histou#url-parameters).
 - URL param to disable Influxdb lookup. Take a look at the [README](https://github.com/Griesbacher/histou#url-parameters).
 
@@ -21,7 +21,7 @@
 
 ## v0.3.10 - 23.11.2016
 ### Features
-- URL/Config param to disable paneltitles for bigger graphs. Url flag: disablePanelTitel
+- URL/Config param to disable paneltitles for bigger graphs. Url flag: disablePanelTitle
 
 ## v0.3.9 - 09.11.2016
 ### Features
diff --git a/histou/basic.php b/histou/basic.php
index 8222bcf..c314063 100644
--- a/histou/basic.php
+++ b/histou/basic.php
@@ -97,8 +97,8 @@ public static function parsArgs()
             }
         }
 
-        if (isset($_GET['disablePanelTitel'])) {
-            static::$disablePanelTitel = true;
+        if (isset($_GET['disablePanelTitle'])) {
+            static::$disablePanelTitle = true;
         }
 
         if (isset($_GET['debug'])) {
